test/datatable.jl

```diff
@@ -341,4 +341,46 @@ module TestDataTable
     @test find(c -> isa(c, NullableCategoricalArray), categorical!(DataTable(A=1:3, B=4:6), :A).columns) == [1]
     @test find(c -> isa(c, NullableCategoricalArray), categorical!(DataTable(A=1:3, B=4:6), [1]).columns) == [1]
     @test find(c -> isa(c, NullableCategoricalArray), categorical!(DataTable(A=1:3, B=4:6), 1).columns) == [1]
+
+    @testset "rename" begin
+        dt = DataTable(A = 1:3, B = 'A':'C')
+        @test names(rename(dt, :A, :A_1)) == [:A_1, :B]
+        @test names(dt) == [:A, :B]
+        @test names(rename!(dt, :A, :A_1)) == [:A_1, :B]
+        @test names(dt) == [:A_1, :B]
+    end
+
+    @testset "size" begin
+        dt = DataTable(A = 1:3, B = 'A':'C')
+        @test_throws ArgumentError size(dt, 3)
+        @test length(dt) == 2
+        @test ndims(dt) == 2
+    end
+
+    @testset "description" begin
+        dt = DataTable(A = 1:10)
+        @test head(dt) == DataTable(A = 1:6)
+        @test head(dt, 1) == DataTable(A = 1)
+        @test tail(dt) == DataTable(A = 5:10)
+        @test tail(dt, 1) == DataTable(A = 10)
+    end
+
+    @testset "misc" begin
+        dt = DataTable(A = 'A':'C')
+        @test sprint(dump, dt) == "DataTables.DataTable  3 observations of 1 variables\n  A: Array{Char}((3,))\n    1: Char A\n    2: Char B\n    3: Char C\n"
```

Comment:
The leading newline is chomped out I believe, so I think you can add a line break between `"""` and `D`.